python/example.py

```diff
@@ -15,11 +15,11 @@
 hash = hashlib.sha256(wasm).hexdigest()
 config = {"wasm": [{"data": wasm, "hash": hash}], "memory": {"max": 5}}
 
-plugin = Plugin(config)
-
-# Call `count_vowels`
-j = json.loads(plugin.call("count_vowels", data))
-print("Number of vowels:", j["count"])
+with Context() as context:
+    with context.plugin(config) as plugin:
```

Comment:
My assumption was that the context_free call would also free up any active plugins in memory. Is that the case? If so you could just use the Context block:

```python
with Context() as context:
  plugin = context.plugin(config)
  j = json.loads(plugin.call("count_vowels", data))
  print("Number of vowels:", j["count"])
```

And when it exits it would free up any plugins you created

Would be the same with the other libraries. For example you'd only need to `defer` the context in go.

Reply:
True, good point! I will update the example